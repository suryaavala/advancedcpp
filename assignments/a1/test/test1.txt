1 2 add
5 sub
6 mult
7 div
8 sqrt

1.0 2.0 add
5.0 sub
6.0 mult
7.0 div
8.0 sqrt

0.1 0.2 add
0.5 sub
0.6 mult
0.7 div
0.8 sqrt

.1 .2 add
.5 sub
.6 mult
.7 div
.8 sqrt

10 20.0 add
50 sub
60.0 mult
70 div
80.0 sqrt