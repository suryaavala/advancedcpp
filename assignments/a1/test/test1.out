2 + 1 = 3
5 - 3 = 2
6 * 2 = 12
7 / 12 = 0
sqrt 8 = 2
2.000 + 1.000 = 3.000
5.000 - 3.000 = 2.000
6.000 * 2.000 = 12.000
7.000 / 12.000 = 0.583
sqrt 8.000 = 2.828
0.200 + 0.100 = 0.300
0.500 - 0.300 = 0.200
0.600 * 0.200 = 0.120
0.700 / 0.120 = 5.833
sqrt 0.800 = 0.894
0.200 + 0.100 = 0.300
0.500 - 0.300 = 0.200
0.600 * 0.200 = 0.120
0.700 / 0.120 = 5.833
sqrt 0.800 = 0.894
20.000 + 10 = 30.000
50 - 30.000 = 20.000
60.000 * 20.000 = 1200.000
70 / 1200.000 = 0.058
sqrt 80.000 = 8.944
